runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -255,6 +267,61 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 		return getSpineSizeWithoutHeader(layout, numberArraylets, dataSize, alignData);
 	}
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	/**
+	 * Checks if arraylet falls into corner case of discontigous data
+	 * Arraylet possible cases:
+	 * 0: Empty arraylets, in this case the array is represented as 
+	 * 		an arraylet however it does not contain any data, but it
+	 * 		does contain an arrayoid (leaf pointer) that points to NULL.
```

Comment:
You could mention here that this is really case of Contiguous Array with length 0. It looks like as Discontiguous because of an implementation